spec/lib/va_profile/contact_information/service_spec.rb

```diff
@@ -32,6 +32,15 @@
           expect(response.person.addresses[0].province).to eq('province')
         end
       end
+
+      it 'has a bad address' do
+        VCR.use_cassette('va_profile/contact_information/person_full', VCR::MATCH_EVERYTHING) do
+          response = subject.get_person
+
+          puts response.person.addresses[1].inspect
```

Comment:
should delete this

Reply:
➕ Looks like a debug line that was forgotten